<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntTestAppMainWindow</class>
 <widget class="QMainWindow" name="AntTestAppMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ANT+ Test Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="searchStickLabel">
      <property name="text">
       <string>Searching for ANT+ USB stick</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="InitializingLabel">
      <property name="text">
       <string>Intializing</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="heartRateLabel">
      <property name="text">
       <string>Heart Rate Sensor</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hrSearchLabel">
         <property name="text">
          <string>Searching..</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hrSensorId">
         <property name="text">
          <string>Unknown ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentHrLabel">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
